### DISCUSSION TITLE
Why does the unit of the y-axis show twice?

### DISCUSSION THREAD

**Discussion by kittipongwangnok** [phoebe-project/phoebe2#956] [General]
I was just wondering about the unit of the y-axis, it shows twice when I plot style='trace'. How can we show the use of the y-axis with only one unit?
![Screenshot 2567-10-08 at 10 38 24 AM](https://github.com/user-attachments/assets/224932b8-9cf6-4998-a4a2-bbdd083f85f5)

**Comment by kecnry** [comment]
I'm not sure - this is not intended and could be a bug or compatibility issue with matplotlib.  What version of matplotlib are you running?  If its later than 3.5.1, can you try downgrading to 3.5.1 and let us know if you get the same results?

We'll try to fix this, but it probably isn't high on the priority list.  If you (or anyone) wants to try to track this down and fix it, I'd be happy to help you open a PR.  In the meantime, you do have access to the figure object and can always modify the labels after plotting with phoebe.

**Comment by kittipongwangnok** [comment]
I already downgraded Matplotlib to version "3.5.1" but I still get the same result as the previous one.

**Reply by kecnry** [reply]
Ok, I'll open [an issue](https://github.com/phoebe-project/phoebe2/issues/958) - thanks for letting us know!
